docs/source/tutorials.rst

```diff
@@ -15,40 +15,217 @@ Prerequisites
 
 Before you begin
 ----------------
-1. **Prepare Your Data**:
 
-   a. Download the `phenotype data <https://raw.githubusercontent.com/mgalardini/2018_ecoli_pathogenicity/master/data/phenotypes/phenotypes.tsv>`_  and save it as specified in :doc:`inputs`.
+1. Install Conda (if NOT already installed):
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   If you don't have Conda installed, you can install it via Miniconda. Miniconda is a minimal installer for Conda.
+
+   a. Download the Miniconda installer:
+
+      For Linux:
+
+      .. code-block:: console
+
+         wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+
+      For macOS:
+
+      .. code-block:: console
+
+         wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
+
+   b. Install Miniconda:
+
+      .. code-block:: console
+
+         bash miniconda.sh -b -p $HOME/miniconda
+
+   c. Initialize Conda:
+
+      .. code-block:: console
+
+         eval "$($HOME/miniconda/bin/conda shell.bash hook)"
+
+   d. Verify the installation:
+
+      .. code-block:: console
+
+         conda --version
+
+   You should see the Conda version printed to the console.
+
+2. Clone the repository using ``git``:
```

Comment:
We also suggest two other installation methods in the documentation, with the first one probably being better for everyday users. Consider changing it to that method or at least mention that they exist